app/src/main/java/com/kickstarter/ui/activities/PaymentMethodsSettingsActivity.kt

```diff
@@ -44,57 +57,79 @@ class PaymentMethodsSettingsActivity : BaseActivity<PaymentMethodsViewModel.View
             paymentResultCallback = ::onPaymentSheetResult
         )
 
-        this.viewModel.outputs.cards()
-            .compose(bindToLifecycle())
-            .observeOn(AndroidSchedulers.mainThread())
-            .subscribe { setCards(it) }
+        compositeDisposable.add(
```

Comment:
Notice that Instead of using `bindToLifecycle())` with RXJava2 every subscription is added to a `compositeDisposable` to be dispose of properly once the Activity is no longer available